### ISSUE TITLE
[Discover Content Feed V1] [Accessibility] - Back button and header and not announced with voice over

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#30614]
## Steps to reproduce
1. Have voice over enabled.
1. Open FF.
1. Tap on “all stories” and listen how the discover more elements are announced.

### Expected behavior
“Back” button and “All stories “header are announced before the first story card.

### Actual behavior
“Back” button and “All stories “header are not announced before the first story card.

### Device & build information
* Device: iPad Air 11' (18.7), iPhone 15 (17.6)
* Build version: v9000(64449)
* First seen version: ?

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-14122)

**Comment by data-sync-user** [comment]
➤ Matt P commented:

No longer need to do this since the stories feed was removed in https://mozilla-hub.atlassian.net/browse/FXIOS-15105 ( https://mozilla-hub.atlassian.net/browse/FXIOS-15105|smart-link )
